### DISCUSSION TITLE
Nested objects

### DISCUSSION THREAD

**Discussion by HenriquezDiego** [mini-software/MiniExcel#666] [Q&A] [answered]
I have a container class that has a property which is another class. This inner class has the properties I want to access in my Excel file. My question is, why is this not working? My template looks something like this: `{{ParentClass.InnerClass.Property}}`

Thank you in advance!

**Comment by michelebastione** [comment, marked as answer]
Support for nested objects is not available yet. You can try out the MappingTemplater from the new fluent api we added in the v2 preview version, it should satisfy your use case.
